### ISSUE TITLE
[TW-811] 'task done' output should include unblocked tasks

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#846]
_Johannes Schlatow on 2011-03-29T18:29:55Z says:_ 

 When I finish a task I get the following output:

<pre>
$ task 1 done
Completed 1 'My task'
Marked 1 task as done.
</pre>

If there is a task 2 that depends on task 1, the user should get a short info about the fact that his action 'unblocked' another task:

<pre>
$ task 1 done
Completed 1 'Blocking task'
Marked 1 task as done.
Unblocked 2 'Follow-up task'
</pre>

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2011-03-29T18:29:55Z
Modified: 2014-02-09T02:06:38Z
```

**Comment by taskwarrior** [comment]
_Cory Donnelly on 2011-03-29T19:22:48Z says:_ 

 Agreed, this would be helpful when we don't break the dependency chain and get notified elsewhere.
